run_test.sh

```diff
@@ -60,7 +61,11 @@ runtest() {
     checkexit
 }
 
-runtest "$(readlink -f $runner)"
+if [[ $PLATFORM == "Linux" ]]; then
+    runtest "$(readlink -f $runner)"
+else
+    runtest "$(greadlink -f $runner)"
```

Comment:
According to https://github.com/Homebrew/homebrew/blob/master/Library/Formula/coreutils.rb#L53 `readlink` should still work on OS X. Can you verify that?